Retrieve also closed issues

Zlopez opened the pull request:
This will add a configurable way to retrieve closed tickets in the last X days. It should catch any tickets that weren't retrieved between runs.

Closes #166

nphilipp commented on jira_sync/repositories/pagure.py:
```diff
@@ -86,10 +87,18 @@ def normalize_issue(self, api_result: dict[str, Any]) -> Issue:
+        if closed:
+            params = {
+                "status": "Closed",
+                "since": arrow.utcnow().shift(days=-self.retrieve_closed_days_ago).int_timestamp,
+            }
```
This would become:

```
import datetime as dt
…
                "since": int(
                    (
                        dt.datetime.now(dt.UTC) - dt.timedelta(days=self.retrieve_closed_days_ago)
                    ).timestamp()
                ),
…
```

nphilipp reviewed (commented):
Looks good to me, but we could do it without adding a 3rd party dependency. 😉

Zlopez commented:
> Looks good to me, but we could do it without adding a 3rd party dependency. 😉

I really like arrow library, but I agree less 3rd party libraries is always better. So I changed it to datetime and removed arrow from the project dependencies.
